indico/modules/events/contributions/controllers/management.py

```diff
@@ -335,6 +338,19 @@ def _process_DELETE(self):
         return jsonify_data(html=_render_subcontribution_list(self.contrib))
 
 
+class RHCreateSubContributionREST(RHManageContributionBase):
+    """REST endpoint to create a subcontribution"""
+
+    def _process_POST(self):
+        form = SubContributionForm(event=self.event_new)
+        if form.validate_on_submit():
+            subcontrib = create_subcontribution(self.contrib, form.data)
+            return jsonify(id=subcontrib.id, contribution_id=subcontrib.contribution_id, event_id=self.event_new.id)
+        response = jsonify_data(form.errors)
```

Comment:
You might be able to `return jsonify_data(form.errors), 400`. Also, I would use `jsonify_data(success=False, errors=form.errors)`